collections/_version2-english/9-fork/4-migration-of-the-objects.md

```diff
@@ -103,15 +107,38 @@ The summary is shown in the following figure:
 {% capture image_src %}{{ "/" | absolute_url }}assets/images/{{page.collection}}/fork/migration-of-the-objects/forking-market.svg{% endcapture %}
 {% include zoom-image.html src=image_src caption="Figure 3. migration of forking market"%}
 
----
-
-*The rest is in progress.*
 # Migration of Non-Forking Markets (not finalized)
 ## Market Itself
+The non-forking markets which are not [finalized]({{glossary_finalized_market}}) can be migrated only to the [winning universe]({{glossary_winning_universe}}). To migrate those markets, you need to make a transaction for that explicitly. It can be done by anyone, not just the [market creator]({{glossary_market_creator}}), and whenever after finalizing the [forking market]({{glossary_forking_market}}). It is one-way and cannot be reversed.
+
+If you migrate the market which is past its [event end time]({glossary_end_time}) in the [parent universe]({{glossary_parent_universe}}), you need to supply [creation bond]({{glossary_creation_bond}}) (which is normally paid by the market creator when creating the market) and submit the [initial report]({{glossary_initial_report}}) in the [winning universe]({{glossary_winning_universe}}) for the market when you migrate it. Because all of the [REP]({{glossary_reputation_token}}) staked on the non-forking markets in the parent universe are refunded to their owner when a fork starts so that they migrate their REP to a [child universe]({{glossary_child_universe}}) during the [forking period]({{glossary_forking_period}}). Therefore, the creation bond is refunded to their owner when a fork starts, and the markets which are past their event end time are reset back to needing the initial report when migrated to a child universe.
```

Comment:
I tried like 6 times to get this paragraph to read smoother but didn't have much luck.  This is the best I could come up with, and it still isn't great.  😢

Reply:
Thanks for the improvement. And I'm feeling guilty for making you tried many times.

> I tried like 6 times to get this paragraph to read smoother but didn't have much luck. 

I believe it's because the original paragraph I wrote isn't good.
What prevents you from reading smoothly? My wording? Order of description? My poor vocabulary? For future reference, I would like to know.